features/Kernel.php

```diff
@@ -130,9 +130,9 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
                 ],
             ],
             'access_control' => [
-                ['path' => '/embedded', 'roles' => 'IS_AUTHENTICATED_FULLY'],
-                ['path' => '/remote', 'roles' => 'IS_AUTHENTICATED_FULLY'],
-                ['path' => '/', 'roles' => 'IS_AUTHENTICATED_ANONYMOUSLY'],
+                ['path' => '/login', 'roles' => 'IS_AUTHENTICATED_ANONYMOUSLY'],
+                ['path' => '/$', 'roles' => 'IS_AUTHENTICATED_ANONYMOUSLY'],
+                ['path' => '/*', 'roles' => 'IS_AUTHENTICATED_FULLY'],
```

Comment:
? why this change?

Reply:
To allow any new feature to be logged-in, except for the homepage & the login page